core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-palette/store/edit-ema-palette.store.ts

```diff
@@ -237,10 +237,11 @@ export class DotPaletteStore extends ComponentStore<DotPaletteState> {
                 filter,
                 page: 40,
                 type: 'WIDGET'
-            })
+            }),
+            this.dotConfigurationService.getKeyAsList('CONTENT_PALETTE_HIDDEN_CONTENT_TYPES')
```

Comment:
Is this a new variable or do we already had that?

Reply:
We already had it for a while